packages/dds/tree/src/test/shared-tree/sharedTreeFuzzTests.ts

```diff
@@ -139,7 +138,7 @@ export async function performFuzzActionsAbort(
 		{
 			edit: async (state, operation) => {
 				const { index, contents } = operation;
-				applyFuzzChange(tree, contents, TransactionResult.Abort);
+				applyFuzzChange(tree, contents, false);
```

Comment:
this one used to Abort, and the one above used to Apply.

Now both "false".

Enums are much better than boolean when its not super clear what they mean. When I'm reading this code, I can't what that boolean does. Enums are more tpye safe, and more readable.

I think it makes sense to reintroduce this enum somewhere and use it in places like this for improved clarity.

Reply:
Thanks, copy paste error - good catch. Yea, I'll see what I can do.